language/functional_tests/tests/testsuite/operators/gen_integer_binary_operators_types_mismatch.py

```diff
@@ -0,0 +1,89 @@
+print("// !!!! generated file, do NOT edit manually !!!!")
```

Comment:
@dariorussi @sblackshear This script was used to generate the test for mismatched types. Does it look good to you if we check it in this way?

Reply:
I think it would probably be preferable to check in the generated code instead of the script.

Reply:
Yeah it did feel a bit weird for me to put a python script there. Got it removed.

However I think in a long term we should still have the script checked in, otherwise this test is unmaintainable. It might be a good idea to rewrite it in rust and invoked as a build script. Let me explore that option.